staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/subresource.go

```diff
@@ -35,10 +35,10 @@ type SubresourceScaler struct {
 }
 
 // ScalePathMappings maps a group/version to the replicas path in a scale subresource
-type ScalePathMappings map[schema.GroupVersion][]interface{}
+type ScalePathMappings map[schema.GroupVersion]fieldpath.Path
 
 // NewSubresourceScaler creates a new SubresourceScaler
-func NewSubresourceScaler(parentEntries []metav1.ManagedFieldsEntry, parentGroupVersion schema.GroupVersion, parentPathPieces []interface{}) (*SubresourceScaler, error) {
+func NewSubresourceScaler(parentEntries []metav1.ManagedFieldsEntry, parentGroupVersion schema.GroupVersion, parentPath fieldpath.Path) (*SubresourceScaler, error) {
```

Comment:
I'm still pretty sure you need to take the full mapping rather than just `parentPath` here.

Reply:
You still haven't addressed that comment.